apicast/src/configuration.lua

```diff
@@ -435,7 +439,8 @@ function _M.download(endpoint)
   if body and res.status == 200 then
     ngx.log(ngx.DEBUG, 'configuration response received:' .. body)
 
-    local ok, err = _M.decode(body)
+    local ok
```

Comment:
I see. If we had the code more divided into methods or used more explicit var names this wouldn't be needed, but I'm OK with that for now.